### ISSUE TITLE
Implement repeater signal locking

### ISSUE THREAD

**Issue by LogicParrot** [cuberite/cuberite#856]
Repeater signal locking is not implemented, this prevents making complex electronics easily. Specifically, Flip-flops are hard without this.

More info about signal locking:
http://minecraft.gamepedia.com/Repeater#Signal_locking

The signal was not expected to pass through the repeater:
![2014-04-06_20 31 31](https://cloud.githubusercontent.com/assets/5030488/2625929/70e38270-bdb1-11e3-80cf-74a5a281ef10.png)

**Comment by jasperweyne** [comment]
Testing right now with Linux x64 build master 407 from the buildserver, it doesn't seem to work anymore?

**Comment by tigerw** [comment]
Yeah, it didn't get re-implemented in the rewrite. It's on the "roadmap".

**Comment by LogicParrot** [comment]
@tigerw :(

**Comment by tigerw** [comment]
Working on it.
